samples/bootloader/prj.conf

```diff
@@ -16,6 +17,10 @@ CONFIG_NRF_RTC_TIMER=n
 CONFIG_CLOCK_CONTROL=n
 CONFIG_FPROTECT=y
 CONFIG_SECURE_BOOT_DEBUG=y
+CONFIG_ASSERT=y
+CONFIG_BL_ROT_VERIFY_ABI_PROVIDE=y
```

Comment:
I have not seen the verb "provide" used in this context. Is this Zephyr nomenclature?

What about _PROVIDED, _ENABLED, _SUPPORT, or _SUPPORTED?

Reply:
How about instead of _PROVIDE and _USE we have _ENABLED and _REQUIRED?

Reply:
Implemented this now. Take a look to see if it works.

Reply:
Yes. Easier to read, for sure... Accepted